core/domain/exp_domain.py

```diff
@@ -6059,7 +6063,10 @@ def _get_property_name_from_content_id(self, content_id: str) -> str:
                     content_id[:10] == 'rule_input')),
         }
 
-        for prop_name, identifier_function in (
+        # We add a 'no branch' flag because items() method will always return.
```

Comment:
This is not because of the property of `.items()` method -- we have defined `property_name_to_content_id_identifier` above and it is not empty. So this line will always return and the branch where is simply exits will not execute.